projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -204,6 +204,7 @@ if (environment.cpq) {
     TrackingFeatureModule,
     VariantsFeatureModule,
     SavedCartFeatureModule,
+    QuickOrderFeatureModule,
```

Comment:
I'm not sure how you're merging things, but make sure you're properly checking the merge conflicts when you merge from develop to your epic.

1 - line 94 under the `if (environment.b2b) {`, SavedCartFeatureModule was never there for a long time. Please remove it
2 - line 93 under the `if (environment.b2b) {`, QuickOrderFeatureModule should be removed as well as it's available for both b2b and b2c

Reply:
![Screen Shot 2021-07-25 at 10 37 15 PM](https://user-images.githubusercontent.com/29099466/126925533-ac8cc789-efe8-4b44-abfa-f1c4d36eda4d.png)